src/audio/kpb.c

```diff
@@ -431,12 +431,17 @@ static int kpb_reset(struct comp_dev *dev)
 
 	trace_kpb("kpb_reset()");
 
+	/* Reset state to be buffering */
+	kpb->state = KPB_STATE_BUFFERING;
 	/* Reset history buffer */
 	kpb->is_internal_buffer_full = false;
 	kpb_clear_history_buffer(kpb->history_buffer);
 	/* Reset amount of buffered data */
 	kpb->buffered_data = 0;
 
+	/* unregister KPB for async notification */
```

Comment:
Maybe start comment from capital letter as other comments inside KPB.

Reply:
sure, let me change it.